tutorials/R/eddy4r/dockerEddy4r.Rmd

```diff
@@ -179,7 +186,11 @@ with both the username and password as **rstudio**. Once complete you are now in
 a RStudio user interface with eddy4R installed and ready to use. 
 
 Additional information about the use of RStudio and eddy4R packages in Docker 
-containers can be found on the <a href="https://github.com/rocker-org/rocker/wiki/Using-the-RStudio-image" target="_blank">rocker-org/rocker website</a> and the <a href="" target="_blank">eddy4RWiki pages</a> [MJ:where is this, I can't find it]. 
+containers can be found on the <a href="https://github.com/rocker-org/rocker/wiki/Using-the-RStudio-image" target="_blank">rocker-org/rocker website</a> and the <a href="" target="_blank">eddy4RWiki pages</a> [MJ:where is this, I can't find it].[CKL: I think this is 
+referring to the extensive wiki in the NEON-FIU-algorithm repo, which 
```

Comment:
"eddy4RWiki pages" -- That is what I was thinking too, but this was from the published paper so I'd hope they would have referenced a public resource.  I'm going to leave in until I can confirm with Dave or Stefan.